src/styles/blocks/page.scss

```diff
@@ -5,3 +5,5 @@
     margin: 0;
   }
 }
+
+$transition-delay: 300ms;
```

Comment:
You can make separate file `variables.scss` for your variables and connect it to main file)

Reply:
agree)